src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label
+        for="bigSearch"
+        class="search__label"
+      >
+        <img
+          src="images/Search.svg"
+          alt="search icon"
+          class="big__icon"
```

Comment:
this class can be just `icon` and styling from `big__icon` should be moved to this class